src/test/java/com/vackosar/gitflowincrementalbuild/mocks/RepoResolver.java

```diff
@@ -35,10 +35,14 @@ public Repository open(DaemonClient client, String name)
     }
 
     private Repository getOrCreateRepo(String name) {
-        if (!repositories.containsKey(name)) {
-            repositories.put(name, createRepo());
+        Repository repository = repositories.get(name);
+        if (repository != null) {
+            repository.incrementOpen();
```

Comment:
Note: Found this method when looking at `org.eclipse.jgit.transport.resolver.FileResolver.open(C, String)`.